server.go

```diff
@@ -2798,34 +2798,24 @@ func (s *server) establishPersistentConnections() error {
 				IdentityKey: nodeAddr.pubKey,
 				Address:     address,
 			}
-			srvrLog.Debugf("Attempting persistent connection to "+
-				"channel peer %v", lnAddr)
-
-			// Send the persistent connection request to the
-			// connection manager, saving the request itself so we
-			// can cancel/restart the process as needed.
-			connReq := &connmgr.ConnReq{
-				Addr:      lnAddr,
-				Permanent: true,
-			}
 
-			s.persistentConnReqs[pubStr] = append(
```

Comment:
since we now call `go s.connectToPersistentPeer(pubStr)` at the end of this func and letting that func handle the what is added to `persistentConnReqs`, we can remove it here. Here we just need to add addresses to `persistentPeerAddrs` so that `connectToPersistentPeer` uses the correct addresses when creating connReqs